### PR TITLE
[Cookbook][cache][varnish] be more precise about version differences

### PR DESCRIPTION
**Pull request by dbu**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | no |
| Applies to | all |
| Fixed tickets | - |

Found this page while trying to figure out what exactly the defaults of varnish are, and realized I was not precise yet.

### CODE REVIEW

**Comment by WouterJ** [comment]
:+1:

File: cookbook/cache/varnish.rst [hunk 1]
```diff
@@ -65,8 +65,9 @@ Ensure Consistent Caching Behaviour
 
 Varnish uses the cache headers sent by your application to determine how
 to cache content. However, versions prior to Varnish 4 did not respect
-``Cache-Control: no-cache``. To ensure consistent behaviour, use the following
-configuration if you are still using Varnish 3:
+``Cache-Control: no-cache``, ``no-store`` and ``private``. To ensure
+consistent behaviour, use the following configuration if you are still
```
Comment (xabbuh) [inline cookbook/cache/varnish.rst]: behavior (we use AE for the docs)

**Comment by dbu** [comment]
fixed spelling for american english (note that behaviour also occurs in 2 documentation pages)

**Comment by xabbuh** [comment]
Thank you @dbu :+1:

**Comment by weaverryan** [comment]
Very nice - the links to the VCL's is very helpful. Thanks!

**Pull request merged**